sdk/simulator-service/src/implementation/simulator-engine/reducer/depositBorrowReducer.ts

```diff
@@ -1,31 +1,28 @@
 import { borrowFromPosition, depositToPosition } from '@summerfi/sdk-common/common/utils'
 import { steps } from '@summerfi/sdk-common/simulation'
-import { addBalance, subtractBalance, getReferencedValue } from '../../utils'
+import { addBalance, subtractBalance, getValueFromReference } from '../../utils'
 import { ISimulationState } from '../../../interfaces/simulation'
 
 export function depositBorrowReducer(
   step: steps.DepositBorrowStep,
   state: ISimulationState,
 ): ISimulationState {
   const afterDeposit = subtractBalance(
-    getReferencedValue(step.inputs.depositAmount),
+    getValueFromReference(step.inputs.depositAmount),
```

Comment:
just a small readability improvement would be to assign `getValueFromReference(step.inputs.depositAmount)` to a variable as it is referenced 2 times, same for `getValueFromReference(step.inputs.borrowAmount)`